Makefile

```diff
@@ -1,11 +1,11 @@
 # requires compiling native extensions with electron support
-NPARAMS=--runtime=electron --target=6.1.2 --disturl=https://atom.io/download/electron
+NPARAMS=--runtime=electron --target=7.1.11 --disturl=https://atom.io/download/electron
 LINUX_VM=ncannasse@virtbuntu
 
 all:
 
 deps:
-	npm install node-gyp -g
+	command -v node-gyp || npm install node-gyp -g
```

Comment:
`command` does not work on Windows, please remove

Reply:
Done. Time for Windows to become a proper OS ;-)